AllReadyApp/Web-App/AllReady/Features/Campaigns/AuthorizedCampaignsQueryHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using AllReady.Models;
+using AllReady.ViewModels.Campaign;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace AllReady.Features.Campaigns
+{
+    public class AuthorizedCampaignsQueryHandler : IAsyncRequestHandler<AuthorizedCampaignsQuery, List<CampaignViewModel>>
+    {
+        private readonly AllReadyContext _context;
+
+        public AuthorizedCampaignsQueryHandler(AllReadyContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<CampaignViewModel>> Handle(AuthorizedCampaignsQuery message)
+        {
+            return await _context.CampaignManagers.Where(c => c.UserId == message.UserId)
```

Comment:
see my comment about if possible, creating a new/different ViewModel to pass back from CampaignController.ManageCampaign() and only querying for the data needed to build the model here instead of constructing a huge ViewModel whose properties might not be used by C:\Projects\allReady\AllReadyApp\Web-App\AllReady\Views\Campaign\ManageCampaign.cshtml